src/Amazon.Lambda.Tools/TemplateProcessor/ITemplateParser.cs

```diff
@@ -91,8 +91,13 @@ public interface IUpdateResourceField
         /// True if the field should contain code like a Lambda package bundle.
         /// </summary>
         bool IsCode { get; }
-        
-        
+
+        /// <summary>
+        /// True if the field points to an ImageUri.
+        /// </summary>
+        bool IsImageUri { get; }
```

Comment:
This name keeps confusing me since it is also matching the name of the property. My instinct is to think the property means the `ImageUri` property is set. What it really means is the `ImageUri` is pointing to already pushed image. Can we rename it to something like `IsImagePushed`?

Reply:
Done.